waggle-dance-integration-tests/src/main/java/com/hotels/bdp/waggledance/WaggleDanceRunner.java

```diff
@@ -174,6 +174,19 @@ public Builder primary(
       return this;
     }
 
+    public Builder primaryWithPrefix(
+        String name,
+        String remoteMetaStoreUris,
+        AccessControlType accessControlType,
+        String... writableDatabaseWhiteList) {
+      checkArgument(isNotEmpty(name));
+      checkArgument(isNotEmpty(remoteMetaStoreUris));
+      primaryMetaStore = new PrimaryMetaStore(name, remoteMetaStoreUris, accessControlType, writableDatabaseWhiteList);
+      primaryMetaStore.setLatency(8000L);
```

Comment:
line 182-185 can maybe be replaced with a call to `primary(name, remoteMetaStoreUris......)`

Reply:
Ah, thank you! That's a great suggestion :)
I've just added it